Assets/General/NodeReflection/Attribute/NodeTagAttribute.cs

```diff
@@ -0,0 +1,48 @@
+﻿using System;
+
+namespace NodeReflection
+{
+    /// <summary>
+    /// Custom attribute for custom node
+    /// </summary>
+    [AttributeUsage(AttributeTargets.Class | AttributeTargets.Struct)]
+    public class NodeTagAttribute
```

Comment:
Shouldn't this attribute de named `NodeAttribute` instead of `NodeTagAttribute`?

Reply:
It could be named this way but since the user will be using the ```Node``` class, the code will look like this :
```
[Node]
public class CustomNode : Node
```
I think it is a bit of a misleading usage like this. However, I agree that ```NodeTag``` is not the best name for the attribute and we could change it to ```NodeInVisualEditor``` or ```VisualNode``` for example.

Same thing for the ```Exposed``` attribute that could be renamed to ```ExposedInEditor``` or ```ExposedInVisualEditor```.